packages/fluentui/react-bindings/src/hooks/useLatest.ts

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { useIsomorphicLayoutEffect } from './useIsomorphicLayoutEffect';
+
+/**
+ * This is mostly useful to get access to the latest value of some props or state inside an asynchronous callback,
+ * instead of that value at the time the callback was created from.
+ *
+ * @param value
+ */
+export function useLatest<T>(value: T): { readonly current: T } {
```

Comment:
also not used in this PR, is the intended use in TMP ?

Reply:
You're correct, removed 👍